Userland/Applications/Screenshoter/ScreenshoterWidget.cpp

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2021, the SerenityOS Developers
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "ScreenshoterWidget.h"
+#include <Applications/Screenshoter/ScreenshoterWindowGML.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/DateTime.h>
+#include <LibCore/File.h>
+#include <LibCore/Timer.h>
+#include <LibCore/StandardPaths.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/Clipboard.h>
+#include <LibGUI/Painter.h>
+#include <LibGUI/Widget.h>
+#include <LibGUI/Window.h>
+#include <LibGUI/WindowServerConnection.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/RadioButton.h>
+#include <LibGUI/SpinBox.h>
+#include <LibGUI/CheckBox.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/Notification.h>
+#include <LibGfx/PNGWriter.h>
+#include <LibGfx/Palette.h>
+#include <serenity.h>
+
+class SelectableLayover final : public GUI::Widget {
```

Comment:
This name is really poor (layover != overlay).
Please call it something like `SelectionOverlay` instead.